File: api_gateway/src/main/java/com/ticketflow/api_gateway/models/client_models/movies_api/ShortMovieModel.java
package com.ticketflow.api_gateway.models.client_models.movies_api;

import java.time.LocalDateTime;

public class ShortMovieModel {
    private Integer id;
    private String title;
    private LocalDateTime startTime;
    private String cinemaHallName;

    public ShortMovieModel(Integer id, String title, LocalDateTime startTime, String cinemaHallName) {
        this.id = id;
        this.title = title;
        this.startTime = startTime;
        this.cinemaHallName = cinemaHallName;
    }

    public Integer getId() {
        return id;
    }

    public String getTitle() {
        return title;
    }

    public LocalDateTime getStartTime() {
        return startTime;
    }

    public String getCinemaHallName() {
        return cinemaHallName;
    }
}

File: backend/dotnet/ticketflow/TicketFlow.ApiGateway/WebApi/MoviesApi/Models/ShortMovieModel.cs
using System;
using TicketFlow.MovieService.Client.Extensibility.Entities;

namespace TicketFlow.ApiGateway.WebApi.MoviesApi.Models
{
    public class ShortMovieModel
    {
        public ShortMovieModel()
        {
        }

        public ShortMovieModel(IMovie movie)
        {
            Id = movie.Id;
            Title = movie.Film.Title;
            StartTime = movie.StartTime;
            CinemaHallName = movie.CinemaHall.Name;
        }

        public int Id { get; set; }

        public string Title { get; set; }

        public DateTime StartTime { get; set; }

        public string CinemaHallName { get; set; }
    }
}